README.md

```diff
@@ -214,7 +214,7 @@ cd llama.cpp
 
 ### Build
 
-In order to build llama.cpp you have three different options.
+In order to build llama.cpp you have different options.
```

Comment:
There are technically still three build scripts - the Makefile, CMakeLists.txt, and build.zig.